pele_platform/Frag/libraries.py

```diff
@@ -30,7 +32,7 @@ def growing_sites(fragment, user_bond):
     return bonds
 
 
-def sdf_to_pdb(file_list, path, logger):
+def sdf_to_pdb(file_list, path, logger, tmpdirname):
```

Comment:
Here I would add a description for this function (and its parameters and returns).

Reply:
Example [here](https://github.com/martimunicoy/peleffy/blob/a81633a105ac87b9e28df403d022c84ca29ed122/peleffy/utils/utils.py#L104)